pkg/controllers/gateway/suite_test.go

```diff
@@ -17,10 +17,10 @@ package gateway
 
 import (
 	"context"
+	"encoding/json"
 	"path/filepath"
 	"testing"
 	"time"
-	"encoding/json"
```

Comment:
@R-Lawton fyi

Reply:
it has to be in alphabetical order?

Reply:
ha yeah, but running `go fmt` will order them for you.